src/sender.js

```diff
@@ -7,15 +7,12 @@ const Pathfinder = require('five-bells-pathfind').Pathfinder
 function Sender (params) {
   this.source_ledger = params.source_ledger
   this.source_username = params.source_username
+  this.source_account = params.source_account || toAccount(this.source_ledger, this.source_username)
   this.source_password = params.source_password
   this.destination_ledger = params.destination_ledger
-  this.destination_username = params.destination_username
```

Comment:
Ya, I don't have any objections to that so long as the corresponding changes are made in fb-demo and fb-visualization.